scripts/ssh_bosh.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -euo pipefail
+
+BOSH_KEY=/tmp/bosh_id_rsa.$RANDOM
+BOSH_IP=$(aws ec2 describe-instances \
+    --filters "Name=key-name,Values=${DEPLOY_ENV}_bosh_ssh_key_pair" \
+    --query 'Reservations[].Instances[].PublicIpAddress' --output text)
+
+trap 'rm -f ${BOSH_KEY}' EXIT
+
+aws s3 cp "s3://gds-paas-${DEPLOY_ENV}-state/bosh_id_rsa" ${BOSH_KEY} && chmod 400 ${BOSH_KEY}
+
+aws s3 cp "s3://gds-paas-${DEPLOY_ENV}-state/bosh-secrets.yml" - | \
+  ruby -ryaml -e 'puts "Sudo password is " + YAML.load(STDIN)["secrets"]["bosh_vcap_password_orig"]'
+echo
+
+# shellcheck disable=SC2086
```

Comment:
This is an interesting one: if we quote it we get a different Shellcheck problem warning us it will be expanded on the remote side.

Reply:
Hmmm... I think it would be better to quote it and disable `SC2029` as it's a more specific check. I'm also not convinced that the check is quite correct in this case as  it only fires when using the `user@host` syntax. It doesn't fire when just specifying a host, even with a variable in there.

Reply:
Done